packages/main/test/pages/CheckBox.html

```diff
@@ -40,6 +40,7 @@
 	<ui5-title>Change Event Test</ui5-title>
 	<ui5-checkbox id="cb1" text="Long long long text"></ui5-checkbox>
 	<ui5-checkbox id="cb2" disabled></ui5-checkbox>
+	<ui5-checkbox id="displayOnlyCb" display-only></ui5-checkbox>
 	<ui5-input id="field"></ui5-input>
```

Comment:
can we add all of its variants for easier theme checking like this

	<ui5-checkbox display-only checked></ui5-checkbox>
	<ui5-checkbox display-only indeterminate checked></ui5-checkbox>

Reply:
That makes sense, Thanks!